components/gantry/multiaxis/multiaxis.go

```diff
@@ -91,8 +91,20 @@ func newMultiAxis(
 	return mAx, nil
 }
 
+// Home runs the homing sequence of the gantry and returns true once completed.
+func (g *multiAxis) Home(ctx context.Context, extra map[string]interface{}) (bool, error) {
+	homed := true
+	for _, subAx := range g.subAxes {
+		homed, err := subAx.Home(ctx, nil)
+		if err != nil {
+			return homed, err
+		}
+	}
+	return homed, nil
```

Comment:
is the intent to have this capture each homed from the for loop? If so, the homed is being shadowed and is not achieving that

Reply:
oops no. fixing now